SparseGrids/tasgridUnitTests.cpp

```diff
@@ -786,4 +793,57 @@ std::vector<std::function<void(void)>> GridUnitTester::getRuntimeErrorCalls() co
     };
 }
 
+bool GridUnitTester::testLAPACKInterface() {
+    bool all_matched = true;
+
+    #ifdef Tasmanian_ENABLE_BLAS
+    // Initialize.
+    const int N = 100;
+    const double a = TasGrid::Maths::pi / 2;
+    const double b = TasGrid::Maths::pi / 4;
+    int M = N;
+    int nsplit = 1;
+
+    std::vector<double> exact_eigs(N);
+    for (int i=0; i<N; i++) {
+        exact_eigs[i] =
+            a + 2.0 * b * std::cos((i+1) * TasGrid::Maths::pi / (N+1));
+    }
+    std::sort(exact_eigs.begin(), exact_eigs.end());
+
+    std::vector<double> D(N, a), E(N-1, b);
+    std::vector<double> W(N), Z(N*N), WORK1(2*N-2), WORK2(4*N);
+    std::vector<int> IBLOCK1(N), ISPLIT1(N), IWORK1(3*N);
+    ISPLIT1[0] = N;
+    IBLOCK1[0] = N;
+
+    // Test LAPACK's dsterf function.
+    TasBLAS::sterf(N, D.data(), E.data());
+    std::sort(D.begin(), D.end());
+    all_matched = all_matched && doesMatch(D, exact_eigs);
```

Comment:
This is correct, but if the test fails you wouldn't know which of the three methods fail.

Add an if-statement:
```
if (not doesMatch(...)){
    std::cout << "ERROR: failed LAPACK test at sterf()\n";
   all_match = false;
}
```

Reply:
Done.